cli/src/main/java/org/opentosca/toscana/cli/commands/AbstractApiCall.java

```diff
@@ -7,12 +7,11 @@
 
 public abstract class AbstractApiCall implements Runnable {
 
-    protected ApiController api;
-
-    @CommandLine.Option(names = {"-v", "--verbose"}, description = "Enable Info Level Process Output")
-    private boolean showVerbose;
     @CommandLine.Option(names = {"-m", "--moreverbose"}, description = "Enable Debug Level Process Output")
-    private boolean showMVerbose;
+    public boolean showMVerbose;
+    @CommandLine.Option(names = {"-v", "--verbose"}, description = "Enable Info Level Process Output")
```

Comment:
What is printed if I use the verbose flag?

Reply:
Verbose -v gives you a "shorter" output then -m. See here:

![verbose](https://user-images.githubusercontent.com/29202379/38097141-dee39a2a-3374-11e8-92b6-23da6a5594d1.png)